### PR TITLE
Docs : Added docusaurus admonitions to docs

### PR DESCRIPTION
**Pull request by shreedharhegde99**
What : 
     Added docusaurus admonitions to docs.

Why : 
  Regarding #1381 

Hi, @shahednasser I am here again with the new PR as you suggested. Please review the same :smiley:. Thanks.

### CODE REVIEW

**Comment by olivermrbl** [comment]
Can I get you to resolve conflicts? Also, @shahednasser will you have a look at it when time allows for it.

**Comment by shreedharhegde99** [comment]
Hey, @shahednasser I have resolved the conflicts.  Please review the request. Thanks :relaxed: .

File: docs/content/how-to/deploying-on-qovery.md [hunk 1]
```diff
@@ -75,7 +87,11 @@ JTW_SECRET=something_secret_jwt
 COOKIE_SECRET=something_secret_cookie
 ```
 
-> Make sure to use actual secrets in a production environment.
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/deploying-on-qovery.md:90]: Can you change to tip?

File: docs/content/how-to/making-your-store-more-powerful-with-contentful.md [hunk 2]
```diff
@@ -76,19 +76,25 @@ npm run migrate:contentful --file contentful-migrations/update-page-module-valid
 
 To use your new Rich Text module **Content > Page > About**, and click **Add Content > Page**. You will now make use of the new Rich Text module to add some more details about your store. You can write your own text or use the text provided below if you just want to copy/paste.
 
-> ### About Medusa
->
-> Medusa is an open-source headless commerce engine for fast-growing businesses. Getting started with Medusa is very easy and you will be able to start selling online with a basic setup in no time, however, the real power of Medusa starts showing up when you add custom functionality and extend your core to fit your needs.
->
-> The core Medusa package and all the official Medusa plugins ship as individual NPM packages that you install into a Node project. You store and plugins are configured in your medusa-config.js file making it very easy to manage your store as your business grows. Custom functionality doesn't have to come from plugins, you can also add project-level functionality by simply adding files in your `src/` folder. Medusa will automatically register your custom functionalities in the bootstrap phase.
+:::note About Medusa
```
Comment (shahednasser) [inline docs/content/how-to/making-your-store-more-powerful-with-contentful.md:79]: Can you change to info (but keep the title)?

File: docs/content/how-to/notification-api.md [hunk 3]
```diff
@@ -77,9 +77,12 @@ class MyService extends NotificationService {
 
 export default MyService;
 ```
+:::note Note
```
Comment (shahednasser) [inline docs/content/how-to/notification-api.md:80]: It's not necessary to add the title "Note" here

File: docs/content/how-to/uploading-images-to-minio.md [hunk 4]
```diff
@@ -46,7 +46,11 @@ Then configure your `medusa-config.js` to include the plugin alongside the requi
 
 The two access keys in the options are the ones created in the previous section.
 
-> Make sure to use an environment variable for the secret key in a live environment.
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/uploading-images-to-minio.md:49]: Can you change to tip?

File: docs/content/how-to/uploading-images-to-s3.md [hunk 5]
```diff
@@ -78,7 +78,11 @@ https://[bucket].s3.[region].amazonaws.com
 
 The two access keys in the options are the ones created in the previous section.
 
-> Make sure to use an environment variable for the secret key in a live environment.
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/uploading-images-to-s3.md:81]: Can you change to tip?

File: docs/content/how-to/uploading-images-to-spaces.md [hunk 6]
```diff
@@ -41,7 +41,11 @@ Then configure your `medusa-config.js` to include the plugin alongside the requi
 
 In the above options, a `spaces_url` is included. This can be found in your Space overview. The `bucket` should point to the name you gave your Space. The `endpoint` identifies the region in which you created the Space. And finally the two keys are the ones created in the previous section.
 
-> Make sure to use an environment variable for the secret key in a live environment.
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/uploading-images-to-spaces.md:44]: Can you change to tip?

File: docs/content/quickstart/quick-start.md [hunk 7]
```diff
@@ -50,7 +50,11 @@ We have created two starters for you that can help you lay a foundation for your
 - [Nextjs Starter](https://github.com/medusajs/nextjs-starter-medusa)
 - [Gatsby Starter](https://github.com/medusajs/gatsby-starter-medusa)
 
-> Note: Medusa runs on port 9000 by default and the storefront starters are both configured to run on port 8000. If you wish to run your storefront starter on another port you should update your CORS settings in your project's `medusa-config.js`.
+:::note
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start.md:53]: Can you change to tip?

File: docs/content/troubleshooting/transaction-error-in-checkout.md [hunk 8]
```diff
@@ -34,7 +34,11 @@ module.exports = {
 }
 ```
 
-> When changing from SQLite to Postgres, you should seed the database again using:
+:::note
+
+When changing from SQLite to Postgres, you should seed the database again using:
+
+:::
```
Comment (shahednasser) [inline docs/content/troubleshooting/transaction-error-in-checkout.md:41]: This should be after the command

File: docs/content/troubleshooting/transaction-error-in-checkout.md [hunk 9]
```diff
@@ -34,7 +34,11 @@ module.exports = {
 }
 ```
 
-> When changing from SQLite to Postgres, you should seed the database again using:
+:::note
```
Comment (shahednasser) [inline docs/content/troubleshooting/transaction-error-in-checkout.md:37]: Can you change to tip?

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for all the work @shreedharhegde99

**Comment by shreedharhegde99** [comment]
Hey @shahednasser , I will implement the changes as you suggested. Thanks.

File: docs/content/advanced/backend/payment/frontend-payment-flow-in-checkout.md [hunk 10]
```diff
@@ -2,11 +2,17 @@
...
+
 
 ```bash npm2yarn
 npm install @medusajs/medusa-js
 ```
+:::note
+
+All code snippets in the following guide, use the JS SDK distributed through **npm**. To install it, run:
+
+:::
```
Comment (shahednasser) [inline docs/content/advanced/backend/payment/frontend-payment-flow-in-checkout.md:15]: I think you misunderstood this. I need you to put inside the note both the text and the code block. The code block should be after the text inside the note.

File: docs/content/how-to/headless-ecommerce-store-with-gatsby-contentful-medusa.md [hunk 11]
```diff
@@ -4,7 +4,11 @@ title: Create a headless ecommerce store with Gatsby, Contentful & Medusa
 
 # Creating a headless ecommerce store with Gatsby, Contentful and Medusa
 
-> Medusa is an open source headless commerce engine that allow you to create amazing digital commerce experiences. Medusa is highly customizable, allowing you to extend the core to fit your needs.
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/headless-ecommerce-store-with-gatsby-contentful-medusa.md:7]: Add the title "About Medusa" and change to "info"

File: docs/content/troubleshooting/transaction-error-in-checkout.md [hunk 12]
```diff
@@ -34,8 +34,11 @@ module.exports = {
 }
 ```
 
-> When changing from SQLite to Postgres, you should seed the database again using:
-
 ```bash npm2yarn
 npm run seed
 ```
```
Comment (shahednasser) [inline docs/content/troubleshooting/transaction-error-in-checkout.md:39]: The command should be inside the note and after the text in the note

File: docs/content/advanced/backend/payment/frontend-payment-flow-in-checkout.md (hunk 10 above)
Comment (shreedharhegde99) [reply docs/content/advanced/backend/payment/frontend-payment-flow-in-checkout.md:15]: Sorry for the misunderstanding. I will correct it. Thanks..

File: docs/content/how-to/notification-api.md [hunk 13]
```diff
@@ -77,9 +77,12 @@ class MyService extends NotificationService {
 
 export default MyService;
 ```
+:::note
```
Comment (shahednasser) [inline docs/content/how-to/notification-api.md:80]: Can you add a new empty line before this?

File: docs/content/troubleshooting/transaction-error-in-checkout.md (hunk 9 above)
Comment (shahednasser) [reply docs/content/troubleshooting/transaction-error-in-checkout.md:37]: You still need to change this @shreedharhegde99

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thank you for your work @shreedharhegde99 !

**Comment by shreedharhegde99** [comment]
> Thank you for your work @shreedharhegde99 !

Thank you @shahednasser for this opportunity and the guidance. Will look forward to more opportunities to contribute.

**Pull request merged**